packages/muelu/test/maxwell/Maxwell3D.cpp

```diff
@@ -220,7 +220,7 @@ int MainWrappers<Scalar,LocalOrdinal,GlobalOrdinal,Node>::main_(Teuchos::Command
     tm = Teuchos::null;
 
     if (solverName == "Belos") {
-        auto tm2 = TimeMonitor(*TimeMonitor::getNewTimer("Maxwell: 2 - Build Belos solver etc"));
+      auto tm2  = TimeMonitor::getNewTimer("Maxwell: 2 - Build Belos solver etc");
```

Comment:
I don't think this would work like you think -- getNewTimer doesn't start the retrieved Teuchos::Time object, it just gets a pointer to it.  The result is that you wouldn't end up timing the thing.  You would just get a zero time.

Reply:
That call syntax is used in other places in the same file.  Not saying it works.  But it does compile.